README.md

```diff
@@ -65,9 +65,10 @@ It is also possible to build a minimal image with:
 ```$ make minimal```
 
 The `minimal` image doesn't have a frontend to update the user network proxy
-(`rush`) settings (see example below). Instead, one needs to use the `QOS_SPEC`
-environment variable to point to a configuration file. One way to add that
-configuration file is by creating a new image based on the `minimal` image.
+(`rush`) settings (see example below). Instead, one needs to create a
```

Comment:
If one uses the `minimal` image and does not provide a QoS config file, one will be generated in the setup.sh script though, correct?

Reply:
Yes, that's right. I just added a comment about that.